deps/wazuh_testing/wazuh_testing/scripts/data_visualizations.py

```diff
@@ -20,6 +20,8 @@ def get_script_arguments():
                         help=f'Base name for the images. Default {None}.')
     parser.add_argument('-c', '--columns', dest='columns', default=None,
                         help=f'Path to Json with Columns to Plot. Default {None}.')
+    parser.add_argument('-u', '--unify', dest='unify', default=False,
+                        help=f'Unify data of the binary processes with their subprocesses to plot. Default {False}.')
```

Comment:
Currently this require to provide a true/false string value. 

It's more usable using store_true action 

```suggestion
    parser.add_argument('-u', '--unify', dest='unify', default=False, action='store_true',
                        help=f'Unify data of the binary processes with their subprocesses to plot. Default {False}.')
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5500/commits/ee85a1dc18d61450298897d04af637de0c4f2051